Remotion/ObjectBinding/Web.UnitTests/UI/Controls/Validation/ValidationFailureHandlingContextTest.cs

```diff
@@ -0,0 +1,57 @@
+﻿// This file is part of the re-motion Core Framework (www.re-motion.org)
+// Copyright (c) rubicon IT GmbH, www.rubicon.eu
+//
+// The re-motion Core Framework is free software; you can redistribute it
+// and/or modify it under the terms of the GNU Lesser General Public License
+// as published by the Free Software Foundation; either version 2.1 of the
+// License, or (at your option) any later version.
+//
+// re-motion is distributed in the hope that it will be useful,
+// but WITHOUT ANY WARRANTY; without even the implied warranty of
+// MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE. See the
+// GNU Lesser General Public License for more details.
+//
+// You should have received a copy of the GNU Lesser General Public License
+// along with re-motion; if not, see http://www.gnu.org/licenses.
+// using NUnit.Framework;
+//
+using Moq;
+using NUnit.Framework;
+using Remotion.ObjectBinding.Web.UI.Controls.BocListImplementation;
+using Remotion.ObjectBinding.Web.UI.Controls.Validation;
+
+namespace Remotion.ObjectBinding.Web.UnitTests.UI.Controls.Validation
+{
+  [TestFixture]
+  public class ValidationFailureHandlingContextTest
+  {
+    private Mock<IBocList> _bocListStub;
+
+    [SetUp]
+    public void Setup ()
+    {
+      _bocListStub = new Mock<IBocList>();
+    }
+
+    [Test]
+    public void ReportErrorMessage_WithEmptyMessage_EmptyMessageSaved ()
```

Comment:
Ah, I see you're passing empty as an expected case. see other comment on Context itself. Not sure if Empty is a good idea / helpful? my goto would be "no".